ivy/functional/backends/paddle/experimental/losses.py

```diff
@@ -0,0 +1,52 @@
+# global
+from typing import Optional
+import paddle
+import paddle.nn.functional as F
+
+# local
+from ivy.func_wrapper import with_unsupported_device_and_dtypes
+from . import backend_version
+
+
+unsupported_dtypes = [
+    paddle.int8,
+    paddle.int16,
+    paddle.int32,
+    paddle.int64,
+    paddle.uint8,
+    paddle.float16,
+    paddle.complex64,
+    paddle.complex128,
+    paddle.bool,
+]
+
+
+@with_unsupported_device_and_dtypes(
+    {"2.5.1 and below": {"cpu": ("float16",)}}, backend_version
```

Comment:
The `unsupported_dtypes` will be added here in the decorator.
They should not be specified separately.
Kindly remove the list you added above.